.github/workflows/build-m1-binaries.yml

```diff
@@ -123,12 +133,19 @@ jobs:
           . packaging/pkg_helpers.bash
 
           if [[ $CHANNEL == "test" ]]; then
+            setup_cuda
             setup_base_build_version
+            setup_macos
```

Comment:
Can you please explain why this is needed for release build? What does this step do?

Reply:
Done Cleaned up